### PR TITLE
Add examples for using Basic Auth with the HTTP Proxy.

### PR DESCRIPTION
**Pull request by voutilad**
Adds curl, python, and JavaScript examples for providing SCRAM username and password. Also picked up some whitespace changes. 😬

### CODE REVIEW

File: docs/develop/http-proxy.mdx [hunk 1]
```diff
@@ -104,7 +104,7 @@ Curl is likely already installed on your system. If not, see [curl download inst
   <TabItem value="env-nodejs" label="NodeJS">
 
 :::note
-This assumes you're in the root directory of an existing NodeJS project. See [Use NodeJS with Redpanda](../../develop/guide-nodejs) for one way to get started with a new NodeJS project and examples of connecting to Redpanda with a native library. 
+This assumes you're in the root directory of an existing NodeJS project. See [Use NodeJS with Redpanda](../../develop/guide-nodejs) for one way to get started with a new NodeJS project and examples of connecting to Redpanda with a native library.
```
Comment (Feediver1) [inline docs/develop/http-proxy.mdx:107]:
```suggestion
This is based on the assumption that you're in the root directory of an existing NodeJS project. See [Use NodeJS with Redpanda](../../develop/guide-nodejs) for one way to get started with a new NodeJS project and examples of connecting to Redpanda with a native library.
```

File: docs/develop/http-proxy.mdx [hunk 2]
```diff
@@ -747,9 +747,47 @@ res = requests.delete(
   </TabItem>
 </Tabs>
 
+## Authenticating with HTTP Proxy
```
Comment (Feediver1) [inline docs/develop/http-proxy.mdx:750]:
```suggestion
## Authenticate with HTTP Proxy
```

Comment (Feediver1) [reply docs/develop/http-proxy.mdx:750]: ^^ per RP Style Guide

**Review by Feediver1 — APPROVED** [review]
Please commit the heading and text update suggestions. Thanks for the PR!

**Comment by voutilad** [comment]
> Please commit the heading and text update suggestions. Thanks for the PR!

Committed your suggested changes.

**Pull request merged**